File: fizzbuzz/src/FizzBuzz.java
import static java.lang.System.out;

public class FizzBuzz {
    public static void main(String args[]) {
        for (int i = 1; i <= 30; i++) {
            if (i % 15 == 0) {
                out.println("FizzBuzz");
            } else if (i % 3 == 0) {
                out.println("Fizz");
            } else if (i % 5 == 0) {
                out.println("Buzz");
            } else {
                out.println(i);
            }
        }
    }
};

File: ruby_challenges/src/fizzbuzz.rb
(1..50).each do |number|
  if number % 15 == 0
    puts 'fizzbuzz'
  elsif number % 3 == 0
    puts 'fizz'
  elsif number % 5 == 0
    puts 'buzz'
  else
    puts number
  end
end